crates/miden-proving-service-client/Cargo.toml

```diff
@@ -31,8 +32,10 @@ tonic-web = { version = "0.12", optional = true }
 async-trait = "0.1"
 miden-objects = { workspace = true, default-features = false }
 miden-tx = { workspace = true, default-features = false }
+miden-tx-batch-prover = { workspace = true, default-features = false }
 prost = { version = "0.13", default-features = false, features = ["derive"] }
 thiserror = "2.0"
+tokio = { version = "1.38", optional = true }
```

Comment:
I would explicitly use `default-features = false` and enable the feature we need (should be "sync" I think). I assume this only works right now because we're already enabling the sync feature somwhere else in the crates.

Reply:
Looks like it was enabled by the "rt" feature used in the proving server binary, I just changed it.